ksqldb-engine/src/test/java/io/confluent/ksql/integration/JoinIntTest.java

```diff
@@ -106,7 +106,7 @@ private void shouldLeftJoinOrderAndItems(final String testStreamName,
 
     final String queryString =
             "CREATE STREAM " + testStreamName + " AS "
-                + "SELECT " + orderStreamName + ".ITEMID, ORDERID, ORDERUNITS, DESCRIPTION "
+                + "SELECT " + orderStreamName + ".ITEMID, ORDERID, ORDERUNITS, `DESCRIPTION` "
```

Comment:
This is an interesting side effect I hadn't considered. I imagine this is a very common field name. Let's try to think of another keyword here (maybe `DESCRIBE` ?) wdyt @derekjn @colinhicks

Reply:
Can't this be fixed in the SqlBase.g4?

When working with it in the past, I found issues with reserved keywords. I had to use quotes to allow me use them until I figured it out. The SqlBase.g4 has a `nonReserved` section that allows some statements to treat reserved keywords as not reserved. Look also in the `SyntaxErrorValidator.isKeywordError()` method in case you cannot fix it in the SqlBase.g4